src/views/texts/TextDetails.vue

```diff
@@ -12,7 +12,9 @@
 							v-list-item-subtitle(class="text--primary" v-text="text.uri")
 						v-col(cols="6")
 							div(v-if="project" class="float-right")
-								v-alert(type="success" text dense) Already part of project "{{ project }}"
+								v-btn(color="primary" @click="removeText")
+									v-icon(left) mdi-minus
+									| Remove from project
```

Comment:
lets change this to a trash can icon and add a tooltip that says remove from project. Lets also make the button red.